src/uv.c

```diff
@@ -107,6 +107,7 @@ static int uvStart(struct raft_io *io,
     }
     rv = uv_timer_start(&uv->timer, uvTickTimerCb, msecs, msecs);
     assert(rv == 0);
+
```

Comment:
Spurious newline :-D

Reply:
Oh I thought I cleaned all my "it need to be perfect everywhere" changes)